src/ceph-volume/ceph_volume/devices/lvm/listing.py

```diff
@@ -70,51 +68,31 @@ def direct_report():
     bypasses the need to deal with the class interface which is meant for cli
     handling.
     """
-    _list = List([])
-    # this is crucial: make sure that all paths will reflect current
-    # information. In the case of a system that has migrated, the disks will
-    # have changed paths
-    _list.update()
-    return _list.full_report()
+    return List([]).full_report()
 
 
+# TODO: move somewhere in our codebase where it's more accessible/usable?
+def is_ceph_device(lv):
```

Comment:
Make this a member of `Volume`?

Reply:
AFAIR, I've deleted this method in later commit.

Reply:
Moved it to api/lvm.py for now without extending it right now, If the extension sounds sensible, I'll perhaps create a separate PR.